### DISCUSSION TITLE
How to use playwright-extra-stealth in tests

### DISCUSSION THREAD

**Discussion by azgm07** [berstend/puppeteer-extra#754] [Q&A]
I would like to know how to apply stealth plugin to the standard testing framework (@playwright/test) if that's possible, or use the same features like debug, tracer and use the command "npx playwright test" as usual instead of a separated script.

The problem that I'm trying to solve is that I need to login into a site that has bot protection to run some testing scripts already done.

I'm quite new to typescript and playwright, so I do know if that's possible and how I can use these features to solve a login problem.

**Comment by harishdhande01** [comment]
Hi, did you get any solution on this?

**Reply by azgm07** [reply]
I figure it out a way using playwright-extra (https://www.npmjs.com/package/playwright-extra) but at that moment it still not just that solve my issue with the website bot protection, I figure it out that using firefox-driver instead of chrome-driver I could bypass the verification without headless and then I copy the session token with it and save to the next executions. 

I stopped to develop this application in January 2023, so I don't if it still works that way.

**Reply by gal-checksum** [reply]
@azgm07 what's the way?
